docs/run-api-server/installing.mdx

```diff
@@ -10,8 +10,9 @@ To install Horizon, you have a few choices. You can...
 - install prebuilt binaries [from our repositories](#package-manager) via your package manager if running a Debian-based system,
 - download a [prebuilt release](https://github.com/stellar/go/releases/latest) of Horizon for your target architecture and operation system, or
 - [build Horizon and Stellar Core yourself](#building) from scratch.
+- use [horizon helm chart](https://github.com/stellar/helm-charts/blob/main/charts/horizon/README.md) which internally uses same prebuilt binary in the runtime image.
 
-**The first method is recommended**: Not only do you ensure OS compatibility and dependency management, you'll also install some convenient wrappers that make running Horizon and Stellar Core in their respective environments much simpler.
+**prebuilt binaries method is recommended**: Not only do you ensure OS compatibility and dependency management, you'll also install some convenient wrappers that make running Horizon and Stellar Core in their respective environments much simpler.
```

Comment:
I'm not sure how best to phrase this but another big win with packages is cryptographic verification of the packages and updates. apt has built-in checks to ensure package were not tampered with so users can guarantee that updates come from SDF

Reply:
added mention of these aspects - https://github.com/stellar/stellar-docs/pull/216/files#diff-cf952d95be5052ac3eb42baa625826c3ebf71d59b3d1c85274606eba312e9d48R15